### PR TITLE
Remove Strelka and Mutect2 from somatic variant calling

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Removed:
- Strelka2 and Mutect2(gatk3.8) from germline and somatic variant calling respectively.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by hassanfa** [comment]
Panel: tumor-normal: removed Mutect2 and Strelka2 (both germline and somatic)
image

Panel: tumor-only: removed Mutect2 and Strelka2 germline:
image

File: .github/workflows/pytest_and_coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          py.test --cov-report=xml --cov=BALSAMIC -rsxv tests/*
+
+      # Run Codecov 
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          file: ./coverage.xml
+          flags: unittests
+          fail_ci_if_error: true
```
Comment (ashwini06) [inline .github/workflows/pytest_and_coveralls.yml:59]: :question: Do you also need to include the path to report: `path_to_write_report: ./coverage/codecov_report.txt`

**Review by ashwini06 — COMMENTED** [review]
🌟  The newer implementation of codecov over previous coversall is nice.

File: .github/workflows/pytest_and_coveralls.yml (hunk 1 above)
Comment (hassanfa) [reply .github/workflows/pytest_and_coveralls.yml:59]: The line by line missing coverage could be because of this. I'll check.

Comment (hassanfa) [reply .github/workflows/pytest_and_coveralls.yml:59]: It is there: https://codecov.io/gh/Clinical-Genomics/BALSAMIC/src/refactor%2Fremove_mutect2_gatk38/BALSAMIC/

Master is missing, and I guess that's why we don't get a cov report. I'll merge this to develop and go from there.

**Pull request merged**